Makefile

```diff
@@ -4,7 +4,8 @@ IMAGE_REPO ?= nmstate
 HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
 HANDLER_IMAGE_SUFFIX ?=
 HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME)$(HANDLER_IMAGE_SUFFIX)
-HANDLER_IMAGE ?= $(IMAGE_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)
+IMAGE_TAG ?= "latest"
```

Comment:
Why do we need this? We already have handler_image_full_name, just export the tag with : as its prefix.